runtime/src/active_stats.rs

```diff
@@ -0,0 +1,64 @@
+use std::sync::atomic::{AtomicUsize, Ordering};
```

Comment:
You know me, I like doc comments. Could you add some `//!`-style comments to the top here?